subprojects/core/src/main/java/org/gradle/execution/plan/DefaultExecutionPlan.java

```diff
@@ -590,9 +594,11 @@ private boolean tryAcquireLocksForNode(Node node, WorkerLeaseRegistry.WorkerLeas
             LOGGER.debug("Cannot acquire worker lease lock for node {}", node);
             return false;
             // TODO: convert output file checks to a resource lock
-        } else if (!canRunWithCurrentlyExecutedNodes(node, mutations)) {
+        } else if (!canRunWithCurrentlyExecutedNodes(mutations)) {
             LOGGER.debug("Node {} cannot run with currently running nodes {}", node, runningNodes);
             return false;
+        } else if (doesDestroyNotYetConsumedOutputOfAnotherNode(node, mutations.destroyablePaths)) {
```

Comment:
Can you add a debug message here as well? So every reason has some debug output as before?

Reply:
We have a debug message here, that's why I didn't add one:

https://github.com/gradle/gradle/blob/08eabb685333ef21023a2c73fdef15ca695e378e/subprojects/core/src/main/java/org/gradle/execution/plan/DefaultExecutionPlan.java#L703

Reply:
👍